tests/2d/numpy/nonzero.py

```diff
@@ -0,0 +1,17 @@
+try:
+   from ulab import numpy as np
+except:
+   import numpy as np
+
+array = np.array(range(16)).reshape((4,4))
+boolean_array = array < 3
+
+# np.nonzero() returns a tuple of corresponding row and column 
```

Comment:
Don't leave trivial comments in the test script! These are not even for humans, and there is the documentation, anyway.

Reply:
Hey, I removed all the comments